extensions/wikia/TemplateDraft/TemplateDraftHelper.class.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class TemplateDraftHelper {
+
+	/**
+	 * Checks if the Title object is a Draft subpage of a template
+	 *
+	 * @param Title $title
+	 * @return bool
+	 */
+	public function isTitleDraft( Title $title ) {
+		/**
+		 * TODO: Improve this check (i18n)
+		 */
+		return strpos( $title->getText(), '/Draft' ) !== false;
```

Comment:
Why not just use `return $title->isSubpage() && $title->getSubpageText() === 'Draft';`?